examples/mockups/pricing.html

```diff
@@ -0,0 +1,786 @@
+                      <div class="header">Enhanced full-text search</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">API access</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Unlimited projects & users</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Private Versions</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Live PR preview</div>
+                      <div class="description">Live preview of all pull requests</div>
+                    </div>
+                  </div>
+                  <div class="item disabled">
+                    <div class="content">
+                      <div class="header">Email build failure notifications</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">2 day support response time</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">4 Concurrent builds</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Custom domain</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Content embedding</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Search term analytics</div>
+                      <div class="description">30 days</div>
+                    </div>
+                  </div>
+                  <div class="item">
+                    <div class="content">
+                      <div class="header">Pageview analytics</div>
+                      <div class="description">30 days</div>
+                    </div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div class="extra content">
+              <span class="left floated share">
+                <i class="icon share alternate"></i>
+                Share
+              </span>
+              <a class="right floated contact">
+                Learn more
```

Comment:
I was thinking it would be good to have at least two more marketing pages separating the community and the corporate pricing with further details.

We could even have individual pages for all plans, or just for some plans, if the amount of information we come to have for those plans justifies it.

Reply:
With the exception of an enterprise level contract card, we probably won't have too much need for specific pages for each plan. We do probably want to link around to the product page where we describe each feature in more detail however.

Reply:
Related to linking to a specific plan, we probably also want something similar for linking to a feature card on the product page. I opened #11 to discuss more.